### PR TITLE
Grab restaurants [Pull Request]

### PR DESCRIPTION
**Pull request by ZolayvarE**
The object the client is getting how has two properties.
- A route Array.
- a restaurants array.

### CODE REVIEW

File: src/server/handlers.js [hunk 1]
```diff
@@ -45,22 +45,77 @@ module.exports = {
...
+          // Send a response to the client if all requisite queries have been made.
+          queryCounter++;
+          queryCounter >= stepsArray.length ? res.send(responseObject) : null;
+        }) 
+
+        // Error callback
+        .catch(function (error) {
+          console.log('Yelp returned an error:', error);
+
+          // Send a response to the client if all requisite queries have been made.
```
Comment (cchrispy) [inline src/server/handlers.js:114]: Is this supposed to be here?

Comment (ZolayvarE) [reply src/server/handlers.js:114]: Yep. It is a series of queries. So if one fails, I still want to return the results of all the successful ones.

Comment (cchrispy) [reply src/server/handlers.js:114]: We probably want to clear out the routes array and restaurants for each server request, so the client wouldn't receive all previous results

Comment (cchrispy) [reply src/server/handlers.js:114]: nvm

**Pull request merged**